modules/assim.sequential/inst/WillowCreek/prep.data.assim.R

```diff
@@ -28,9 +28,15 @@ prep.data.assim <- function(start_date, end_date, numvals, vars, data.len = 3, s
   
   #Reading the columns we need
   cols <- grep(paste0("_*_f$"), colnames(gapfilled.vars), value = TRUE)
+<<<<<<< HEAD
+  gapfilled.vars <- gapfilled.vars %>% dplyr::select(Date=date...11, Flag=Flag...10,cols)####goes wrong
+
+ #Creating NEE and LE filled output 
+=======
   gapfilled.vars <- gapfilled.vars %>% dplyr::select(Date=date...11, Flag = Flag...10,cols)
   
   #Creating NEE and LE filled output 
+>>>>>>> c2377c10864b7c149fa4e3613d212fb8f509d906
```

Comment:
This syntax indicates a merge conflict. You need to resolve this manually because git couldn't figure out which version to keep